social_core/tests/backends/test_google.py

```diff
@@ -16,64 +16,23 @@
 
 class GoogleOAuth2Test(OAuth2Test):
     backend_path = 'social_core.backends.google.GoogleOAuth2'
-    user_data_url = 'https://www.googleapis.com/plus/v1/people/me'
+    user_data_url = 'https://www.googleapis.com/oauth2/v3/userinfo'
     expected_username = 'foo'
     access_token_body = json.dumps({
         'access_token': 'foobar',
         'token_type': 'bearer'
     })
     user_data_body = json.dumps({
-        'aboutMe': 'About me text',
-        'cover': {
-            'coverInfo': {
-                'leftImageOffset': 0,
-                'topImageOffset': 0
-            },
-            'coverPhoto': {
-                'height': 629,
-                'url': 'https://lh5.googleusercontent.com/-ui-GqpNh5Ms/'
-                       'AAAAAAAAAAI/AAAAAAAAAZw/a7puhHMO_fg/photo.jpg',
-                'width': 940
-            },
-            'layout': 'banner'
-        },
-        'displayName': 'Foo Bar',
-        'emails': [{
-            'type': 'account',
-            'value': 'foo@bar.com'
-        }],
-        'etag': '"e-tag string"',
-        'gender': 'male',
-        'id': '101010101010101010101',
-        'image': {
-            'url': 'https://lh5.googleusercontent.com/-ui-GqpNh5Ms/'
+        'profile': 'https://plus.google.com/101010101010101010101',
```

Comment:
Where is the profile coming from? I don't get it when I do a GET on the `userinfo` endpoint. I get:
```
{
 "id": "**************",
 "email": "*****@johanneshoppe.com",
 "verified_email": true,
 "name": "Johannes Hoppe",
 "given_name": "Johannes",
 "family_name": "Hoppe",
 "link": "https://plus.google.com/+JohannesHoppe1",
 "picture": "https://lh4.googleusercontent.com/-DkWjGAz2yP8/AAAAAAAAAAI/AAAAAAAAEv4/ro4cfX5UqDs/photo.jpg",
 "gender": "male",
 "locale": "en",
 "hd": "johanneshoppe.com"
}
```

Reply:
Your example response looks like a one coming from a different endpoint than the one used in this PR. https://www.googleapis.com/oauth2/v3/userinfo does return a `profile` field (and has `sub` instead of `id` for example). If you're using the same endpoint, then we're getting different results..

Reply:
Sorry, I used v2. All good:
```JSON
{
  "profile": "https://plus.google.com/+JohannesHoppe1", 
  "family_name": "Hoppe", 
  "sub": "***", 
  "picture": "https://lh4.googleusercontent.com/-DkWjGAz2yP8/AAAAAAAAAAI/AAAAAAAAEv4/ro4cfX5UqDs/photo.jpg", 
  "locale": "en", 
  "email_verified": true, 
  "given_name": "Johannes", 
  "gender": "male", 
  "email": "***@johanneshoppe.com", 
  "hd": "johanneshoppe.com", 
  "name": "Johannes Hoppe"
}
```